### PR TITLE
pdf-view: tell user how to select pdf text when copy_select copy fail

### PR DESCRIPTION
**Pull request by tumashu**

### CODE REVIEW

File: app/pdf-viewer/buffer.py [hunk 1]
```diff
@@ -162,7 +162,7 @@ def copy_select(self):
             self.message_to_emacs.emit(content)
             self.buffer_widget.cleanup_select()
         else:
-            self.message_to_emacs.emit("Cannot copy, you haven't select anything!")
+            self.message_to_emacs.emit("Cannot copy, you should double click left mouse button then drag to select pdf text!")
```
Comment (MatthewZMD) [inline app/pdf-viewer/buffer.py]: you should double click your mouse and hover through the text on the PDF. Don't click and drag!

Comment (MatthewZMD) [reply app/pdf-viewer/buffer.py]: s/Don't/Do not

**Comment by tumashu** [comment]
Fixed, but I think we hade better support two drag mode.

**Comment by tumashu** [comment]
1. 2click + drag  
2. 2click+  hold button drag

**Pull request merged**